### PR TITLE
New responses based on #1044

### PR DESCRIPTION
**Pull request by Akurosia**
Added responses based on #1044

### CODE REVIEW

File: resources/responses.js [hunk 1]
```diff
@@ -203,6 +206,15 @@ let Responses = {
     };
     return obj;
   },
+  bigAoe: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Extreme raid damage!',
```
Comment (quisquous) [inline resources/responses.js]: how about just ... big aoe, to match other triggers?

File: resources/responses.js [hunk 2]
```diff
@@ -240,14 +264,39 @@ let Responses = {
     };
     return obj;
   },
-  stack: (sev) => {
+  stackMiddle: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'aoe',
```
Comment (quisquous) [inline resources/responses.js]: This needs translation?

File: resources/responses.js [hunk 3]
```diff
@@ -571,14 +646,34 @@ let Responses = {
...
+  },
+  moveChainsTogether: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Move chains together',
+      de: 'Ketten zusammen bewegen',
+    };
+    return obj;
+  },
+  earthshacker: (sev) => {
```
Comment (quisquous) [inline resources/responses.js]: earthshacker => earthshaker

File: resources/responses.js (hunk 1 above)
Comment (Akurosia) [reply resources/responses.js]: changed it :)

**Pull request merged**